hack/00-osd-managed-cluster-config.selectorsyncset.yaml.tmpl

```diff
@@ -4906,6 +4906,101 @@ objects:
             annotations:
               message: Layered SRE "{{ $labels.user }}" elevated to layered cluster-admin({{
                 $labels.group }}) more than 2 hours.
+    - apiVersion: monitoring.coreos.com/v1
+      kind: PrometheusRule
+      metadata:
+        labels:
+          prometheus: sre-managed-upgrade-operator-alerts
+          role: alert-rules
+        name: sre-managed-upgrade-operator-alerts
+        namespace: openshift-monitoring
+      spec:
+        groups:
+        - name: sre-managed-upgrade-operator-alerts
+          rules:
+          - alert: UpgradeConfigValidationFailedSRE
+            expr: 'upgradeoperator_upgradeconfig_validation_failed > 0
+
```

Comment:
Why here is an empty line?

Reply:
caused by the format of the expr.
If we write it as 
```
expr: |
 xxxx
```
it will add the line break in the end. 
I can see the format is supported in the cluster.
Fixed as we do not need the expression to be multiple lines.